BM6toXPlaneVideo/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Has this (BM6toXPlaneVideo) been tested?

Reply:
No